Cargo.toml

```diff
@@ -29,8 +29,8 @@ verify = ["dep:geiger"]
 [dependencies]
 current_platform = "0.2.0"
 cfg-if = "1"
-pgx = { version = "0.5.0-beta.1" }
```

Comment:
Nit: `~` doesn't make a difference here from the default (and anything aside from the default hits the "cargo's resolver behaves badly" behavior)

Reply:
Oh, unless this is a hack to make patch.crates-io work? I don't *think* it should matter for that but... maybe?

Reply:
I was mostly experimenting, tbh.